core/src/avm2/globals/flash/system/Capabilities.as

```diff
@@ -1,20 +1,158 @@
+            return false;
+        }
+
+        public static function get hasScreenPlayback():Boolean {
+            stub_getter("flash.system.Capabilities", "hasScreenPlayback");
+            return false;
+        }
+
+        public static function get hasStreamingAudio():Boolean {
+            stub_getter("flash.system.Capabilities", "hasStreamingAudio");
+            return true;
+        }
+
+        public static function get hasStreamingVideo():Boolean {
+            stub_getter("flash.system.Capabilities", "hasStreamingVideo");
+            return true;
+        }
+
+        public static function get hasTLS():Boolean {
+            stub_getter("flash.system.Capabilities", "hasTLS");
+            return true;
+        }
+
+        public static function get hasVideoEncoder():Boolean {
+            stub_getter("flash.system.Capabilities", "hasVideoEncoder");
+            return true;
+        }
+
+        public static function get isDebugger():Boolean {
+            stub_getter("flash.system.Capabilities", "isDebugger");
+            return false;
+        }
+
+        public static function get isEmbeddedInAcrobat():Boolean {
+            stub_getter("flash.system.Capabilities", "isEmbeddedInAcrobat");
+            return false;
+        }
+
         public native static function get language(): String;
+
+        public static function get localFileReadDisable():Boolean {
+            stub_getter("flash.system.Capabilities", "localFileReadDisable");
+            return false;
+        }
+
         public static function get manufacturer(): String {
             stub_getter("flash.system.Capabilities", "manufacturer");
-            return "Adobe Windows"
+            return "Adobe Windows";
+        }
+
+        [API("662")]
+        public static function get maxLevelIDC():String {
+            stub_getter("flash.system.Capabilities", "maxLevelIDC");
+            return "5.1";
         }
-        public static function get isDebugger(): Boolean {
-            return false
+
+        public native static function get os(): String;
+
+        public native static function get pixelAspectRatio():Number;
+
+        public native static function get playerType(): String;
+
+        public static function get screenColor():String {
+            stub_getter("flash.system.Capabilities", "screenColor");
+            return "color";
+        }
+
+        public native static function get screenDPI():Number;
+
+        public native static function get screenResolutionY():Number;
+
+        public native static function get screenResolutionX():Number;
+
+        public static function get serverString():String {
+            stub_getter("flash.system.Capabilities", "serverString");
+            return "A=t&SA=t&SV=t&EV=t&MP3=t&AE=t&VE=t&ACC=f&PR=t&SP=t&SB=f&DEB=t&V=WIN%208%2C5%2C0%2C208&M=Adobe%20Windows&R=1600x1200&DP=72&COL=color&AR=1.0&OS=Windows%20XP&L=en&PT=External&AVD=f&LFD=f&WD=f";
```

Comment:
This says PR=t but now the default `hasPrinting` is `false`

Reply:
Fixed, thanks!